wireless_modules/middle_module/test.py

```diff
@@ -0,0 +1,70 @@
+import paho.mqtt.client as mqtt
```

Comment:
Does this replace the other `test.py` up a directory?

Reply:
We could, I thought of leaving the test.py up a directory for testing Mqtt topics but I just realised we may as well do that with the new test.py sine it's almost the same.